test/probes/restify.test.js

```diff
@@ -30,8 +30,9 @@ if (semver.satisfies(process.version, '>=8.0.0')) {
 
 describe(`probes.restify ${pkg.version}`, function () {
   let emitter
-  let fsState
+  let fsState;
```

Comment:
well, the previous line could also end in `;` in this case

Reply:
yes, it could have eagle-eye. i would really like to go through module by module and make that change with *no* other changes.